flatten-data-ghcn_d.ipynb

```diff
@@ -0,0 +1,1167 @@
+{
```

Comment:
<p>A couple of notes:</p><p>1) Is this use for showcasing? Do we need to print out all the tables or can we just show a few for illustrative purposes?</p><p>2) This can be accomplished by a simple list comprehension</p><p> </p><pre class="ql-syntax" spellcheck="false"># Iterate over all tables in the schema and returns only those that have a year as the last four characters
tables_used = [table.table_id for table in tables if int(table.table+id[-4:]) &gt; 1900]
</pre><p>&nbsp;&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>The reasoning behind this is to illustrate a for loop over the big query tables. In a later cell the print function is going to be replaced by a query so essentially we are doing one query per year table.</p><p> </p><p>Thank you for showing me a new technique of a list comprehension. I will test it around a bit and see if I can do the same thing with this technique.</p><p> </p>